src/auth-service/utils/favorite.util.js

```
@@ -123,72 +170,76 @@ const favorites = {
         firebase_user_id: firebase_user_id,
       };
 
+      // Get current favorites for the user
       let unsynced_favorite_places = (
         await FavoriteModel(tenant.toLowerCase()).list({ filter }, next)
       ).data;
 
+      // Remove _id for comparison
       unsynced_favorite_places = unsynced_favorite_places.map((item) => {
         delete item._id;
         return item;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)